index.html

```diff
@@ -10,80 +10,27 @@
         <meta name="viewport"           content="width=device-width, initial-scale=1.0">
         <meta name="theme-color"        content="#1e1e1e">
 
-        <!-- Include these early and directly so they happen first -->
-        <script>
-            // Install Cascade Studio as a Progressive Web App for Offline Access
-            // This needs to be put before ANY HTTP Requests are made, so it can cache them.
-            if ('serviceWorker' in navigator) {
-                navigator.serviceWorker.register('service-worker.js').then(function(registration) {
-                    registration.update(); // Always update the registration for the latest assets
-                }, function() {
-                    console.log('Could not register Cascade Studio for offline use!');
-                });
-            } else {
-                console.log('Browser does not support offline access!');
-            }
-
-            // Begins loading the CAD Kernel Web Worker
-            if (window.Worker) {
-                var cascadeStudioWorker = new Worker('./js/CADWorker/CascadeStudioMainWorker.js');
-                // Ping Pong Messages Back and Forth based on their registration in messageHandlers
-                var messageHandlers = {};
-                cascadeStudioWorker.onmessage = function (e) {
-                    if(e.data.type in messageHandlers){
-                        let response = messageHandlers[e.data.type](e.data.payload);
-                        if (response) { cascadeStudioWorker.postMessage({ "type": e.data.type, payload: response }) };
-                    }
-                }
-            }
```

Comment:
Worker init code has been moved into it's own file `js/MainPage/CascadeWorkerInit.js`, the worker is then imported into `CascadeMain` and `CascadeView`